project-incubation-exit.md

```diff
@@ -96,6 +96,17 @@ decision process by informing its members of key aspects of the project.
     -   Project is set up with Continuous Integration
     -   All project repos have implemented the [common repository structure](./repository-structure.md)
 
+-   Security
+
+    The project must identify key contact points, also called as
```

Comment:
Since security mavens was a term specifically used in the past for Hyperledger staff members, I would recommend changing this and the following line to:
```
The project must identify key contact points to address security related questions and concerns.
```

Reply:
Removed the use of "maven" term. Any suggestions on what can we name key contact points from each project?